driver-async/src/main/com/mongodb/async/client/MongoClients.java

```diff
@@ -38,7 +40,6 @@
  * @since 3.0
  * @deprecated Prefer the Reactive Streams-based asynchronous driver (mongodb-driver-reactivestreams artifactId)
  */
-@SuppressWarnings("deprecation")
```

Comment:
Why this change?

Reply:
I started by deleting all the method-level annotations below since this one should cover, then realized there aren't any compiler warnings for this class anyway, so just removed all of them.  Maybe that's due to a change in build.gradle though (didn't we turn off all deprecation warnings?)  Happy to revert this.